pullrequest.rst

```diff
@@ -238,6 +238,8 @@ The automated patch checklist runs through:
 * Has the documentation been updated?
 * Has the test suite been updated?
 * Has an entry under ``Misc/NEWS.d/next`` been added?
+  (using `blurb-it <https://blurb-it.herokuapp.com/>`_,
+   or the `blurb <https://pypi.org/project/blurb/>`_ tool)
```

Comment:
CI complains:
```
/home/travis/build/python/devguide/pullrequest.rst:242:Unexpected indentation.
```
I think this should do it:
```suggestion
  or the `blurb <https://pypi.org/project/blurb/>`_ tool)
```

Reply:
thanks